.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+     
+      - name: Install Hunspell
+        run: |
+           sudo apt-get update
+           sudo apt-get install hunspell
+           
+      - name: Run cd scripts; make -k
+        run: cd scripts; make -k
```

Comment:
Why not just use `make -C scripts -k` and not need to `cd` and then the next step isn't needed.

Reply:
I prefer to keep it the way it is. It's the same algorithm as in `.travis.yml`. 🙂